kuma/demos/templates/demos/listing_tag.html

```diff
@@ -19,7 +19,13 @@
       <header id="page-head" class="column-container">
         <div class="column-8">
             <div class="main">
-                <h1 class="page-title"><img src="{{ static('img/icons/' + tag.name.replace('tech:','') + '.png') }}" alt="" width="160" height="160"> {{ tag_title(tag) }}</h1>
+                <h1 class="page-title">
+                    {# Don't attempt to load a technology icon on Dev Derby contest pages #}
+                    {% if not 'challenge:' in tag.name %}
```

Comment:
Mind using `{% if 'challenge:' not in tag.name %}` here? That would make it only one boolean evaluation instead of two.